.testcaferc-with-video.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Would it be possible to create an `npm` script to be able to run with this config more easily?